contents/docs/user-guides/data-management.mdx

```diff
@@ -0,0 +1,92 @@
+
+Event definitions differ from actions in that a definition can represent exactly one type of event, whereas actions are composed of multiple matching groups and therefore multiple events. We recommend using actions if your data managing use case cannot be solved with simple definitions.
+
+### Event Property Definition
+
+An event property definition is a mutable representation an event property that is nested inside an event. Event property definitions are globally defined, meaning that the same properties shared across events will share one definition.
+
+### Definition Fields
+
+Today, you can add descriptions and tags to both events and event property definitions. Event definitions can also be verified which tells collaborators that an event should be favored over other similar events.
+
+### Creating definitions
+
+Before creating your first definition, make sure at least one event of that type has been sent to PostHog. You can use the "Live Events" page to observe that your event has been ingested properly.
+
+## Creating a definition from a filter
+
+Definitions can be created on the fly anywhere in the app where there is a filter.
+
+Open the filter and hover over the item you'd like to create a definition for. See that a definition pops up on the side.
+
+![PostHog - definition card view](../../images/features/data-management/definition-card-view.png)
+
+This card represents the definition that has already been created. If you haven't created a definition here, PostHog will display an empty definition you can fill out here.
+
+Click "Edit" in the definition card.
+
+![PostHog - definition card edit](../../images/features/data-management/definition-card-edit.png)
+
+From here you can add some context that captures some context about this event or property. Keep in mind this definition will persist everywhere this event is displayed in the app.
+
+Once you've finalized everything, save your definition, and you're good to go!
+
+## Creating a definition from the Data Management page
+
+If you'd like one central place to manage all your definitions, go to the Data Management page to see a flat list of event and event property definitions.
+
+![PostHog - data management page](../../images/features/data-management/data-management-page.png)
+
+From here you may hover over event and event property names to edit definitions.
+
+You may also explore all properties nested under each event to quickly understand how your data is structured.
+
+![PostHog - data management expand](../../images/features/data-management/data-management-event-expand.png)
+
+## How to use Data Management as a team (best practices)
+
+To help you in your journey to data management nirvana, here are a few rules of thumb to consider when using these new features.
+
+1. The more definitions you add, the easier it will be for your teammates to understand what each event and property represents.
```

Comment:
```suggestion
**1. The more definitions you add, the easier it will be for your teammates to understand what each event and property represents.**
```